docker-compose.yml

```diff
@@ -34,7 +34,15 @@ services:
                 "auth-orcid": {
                     "clientID": "${ORCID_CLIENT_ID}",
                     "clientSecret": "${ORCID_CLIENT_SECRET}"
+                },
+                "meca": {
+                  "s3": {
+                    "params": {
+                      "Bucket": "${S3_BUCKET}",
```

Comment:
why is this Uppercase?

Reply:
indentation also seems to be off wrt the rest of the JSON dictionary

Reply:
usually `,` at the end of the last value is supported in JavaScript literals, but issn't in JSON; is that a problem?

Reply:
`aws-sdk` likes it like that :(